### PR TITLE
refactors local secret, adds env var imp dapr issue #1961

### PR DESCRIPTION
**Pull request by mchmarny**
# Description

Refactors local secrets (used for development) into `local.file` and adds `local.env` implementation. 

## Issue reference

[dapr issue #1961](https://github.com/dapr/dapr/issues/1961)

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Created docs issue in the https://github.com/dapr/docs/pull/766

### CODE REVIEW

File: secretstores/local/env/envstore_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import (
+	"os"
+	"testing"
+
+	"github.com/dapr/components-contrib/secretstores"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+const secretValue = "secret"
```
Comment (amanbha) [inline secretstores/local/env/envstore_test.go:16]: This is not used.

**Comment by amanbha** [comment]
@mchmarny  LGTM, Could you please open a doc issue to add docs for it.

File: secretstores/local/env/envstore_test.go (hunk 1 above)
Comment (mchmarny) [reply secretstores/local/env/envstore_test.go]: fixed

**Comment by mchmarny** [comment]
Added pointer to doc PR

**Pull request merged**